faq/vpc.mdx

```diff
@@ -9,7 +9,28 @@ hero: assets/private-networks.webp
 ---
 
 <Message type="note">
-  This page concerns the **VPC** / **Regional Private Networks** product currently in [Private Beta](https://www.scaleway.com/en/betas/#regional-private-network) and accessible only to approved beta testers. If you're looking for the Public Gateways or classic Private Networks FAQ, note that this content has been relocated:
+  This page concerns the **VPC** / **regional Private Networks** product currently in Public Beta. If you're looking for the Public Gateways or classic Private Networks FAQ, note that this content has been relocated:
   - [Public Gateways FAQ](/faq/public-gateways/)
   - [Private Networks FAQ](/faq/private-networks/)
-</Message>
\ No newline at end of file
+</Message>
+
+## VPC
+
+### What is the difference between VPC and a regional Private Network ?
+One default VPC (**V**irtual **P**rivate **C**loud) for every available region is automatically created in each Scaleway [Project](/console/my-project/concepts/#project). A VPC offers layer 3 isolation. Currently, at Public Beta stage, VPC is only available for the Paris region.
+
+A regional Private Network sits inside a VPC, and lets you connect resources across multiple AZs within the VPC's region, to a Private Network. This network offers layer 2 isolation. You can create multiple regional Private Networks within a VPC.
```

Comment:
not sure if I'm misreading, but this sentence gives the impression that a regional private network is connecting somehow other private networks, which is is not the case. 

Instead we can just say that a regional private network is a layer 2 network (as private networks), but they are spawned across all the AZs of the region (while private networks are scoped to the AZ)